script/mail_new_activity.rb

```diff
@@ -109,13 +109,14 @@ def story_subject(story, prefix = "")
 
         if s.description.present?
           body.push s.description.to_s.word_wrap(EMAIL_WIDTH)
+          body.push ""
         end
 
-        if s.url.present?
-          if s.description.present?
-            body.push ""
-          end
+        body.push "Vote: #{Routes.story_short_id_url s}"
+        body.push "-- "
```

Comment:
```suggestion
```

I know some of the textmode email clients collapse/hide sigs by default, and I don't want to hide the entire story text. Their use is overrepresented among our cohort of older coders.)